hazelcast/src/main/java/com/hazelcast/internal/partition/operation/MigrationCommitOperation.java

```diff
@@ -48,6 +50,11 @@ public MigrationCommitOperation(PartitionRuntimeState partitionState) {
 
     @Override
     public void run() {
+        NodeEngine nodeEngine = getNodeEngine();
+        if (!nodeEngine.isRunning()) {
```

Comment:
I see usages with `nodeEngine.getNode().getState() == NodeState.SHUT_DOWN` . Are they equivalent? nodeEngine.isRunning and nodeState?